NSS-Day-Cohort-49/tabloid-full-stack-spicy-sweet-chilis
# Kk tag crud

#### Changes Made
1. Added files to client component "tag": Tag.js, TagForm.js, TagList.js
2. Added file to client modules: tagManager.js
3.  Modified client files :  ApplicationViews.js, Header.js  
4. Added files to server side: Tag.cs, TagRepository.cs, ITagRepository.cs, TagController.cs
5. Modified Startup.cs: added service for ITagRepository

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout kk-tag-Crud
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > Running Swagger, test CRUD for tags plus GetById
    >In browser, select Tag Manager in menu, and view tag list in new render
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
